app/javascript/controllers/version_navigation_controller.js

```diff
@@ -0,0 +1,11 @@
+import { Controller } from 'stimulus'
+
+export default class extends Controller {
+  connect () {
+    // auto-scroll to the active version so it's visible on initial page load
+    const versionNavigation = this.element
+    const activeVersion = versionNavigation.querySelector('.active')
```

Comment:
It's worth mentioning, but probably _not_ worth changing, that we'll often pass in class names where the controller is initialized. Ex:

``` html
<div data-controller="version-navigation" data-version-naviation-active-value="active">
```

And then reference the value in the controller:
``` js
static values = { active: String }

connect () {
  const versionNavigation = this.element
  const activeVersion = versionNavigation.querySelector(this.activeValue)
}
```

It also looks like this has changed slightly in Stimulus 2.0. I think you're contribution here is the first 2.0 code. Everything prior to this was 1.x. But, this is all just FYI. I think having `.active` in there directly is fine.